test/src/pose.spec.js

```diff
@@ -58,11 +58,36 @@ const MATRIX_TEST_CASES = [{
   }
 ];
 
+function arePosesEqual(pose1, pose2) {
+  return equals(pose1.position, pose2.position) & equals(pose1.orientation, pose2.orientation);
```

Comment:
Most of our math objects support `.equals` and `.exactEquals`. While they get it automatically through inheritance, maybe we could try to maintain the same API?

https://github.com/uber-web/math.gl/blob/master/src/lib/math-array.js#L72